### ISSUE TITLE
Error when opening a sidebar

### ISSUE THREAD

**Issue by lukamanitta** [noib3/nvim-cokeline#47]
I'm using this rendering configuration
```
    rendering = {
        max_buffer_width = 20,

        left_sidebar = {
            filetype = 'NvimTree',
            components = {
                text = 'NvimTree',
                hl = {
                    fg = get_hex('Normal', 'fg'),
                    bg = get_hex('NvimTreeNormal', 'bg'),
                    style = 'bold',
                },
            },
        },
    },
```

Whenever I open NvimTree (with NvimTreeToggle), I get the below error.

```
E5108: Error executing lua ...ack/packer/start/nvim-cokeline/lua/cokeline/sidebars.lua:46: attempt to index a nil value
stack traceback:
	...ack/packer/start/nvim-cokeline/lua/cokeline/sidebars.lua:46: in function 'get_sidebar_components'
	...ack/packer/start/nvim-cokeline/lua/cokeline/sidebars.lua:85: in function 'get_left_right_sidebar_components'
	...ck/packer/start/nvim-cokeline/lua/cokeline/rendering.lua:186: in function <...ck/packer/start/nvim-cokeline/lua/cokeline/rendering.lua:172>
```

The error is coming from the second of these lines:

```
    components[#components].text = components[#components].text
      .. str_rep(" ", space_left)
```

After this error is dismissed the bufferline goes back to default, but works as configured before the error is triggered.

**Comment by noib3** [comment]
Should be fixed in 0.3.0. Note that the sidebar api has changed, see the CHANGELOG for details. Let me know if you still have issues.

**Comment by lukamanitta** [comment]
Seems to work now, and the consistency with the other components field is very nice!
